docs/updates/2019-01.md

```diff
@@ -0,0 +1,44 @@
+# January 2019
+
+New year, new features, new solutions. In this update we will share some news about our latest tool releases.
```

Comment:
"In this update" feels a bit weird if you think about people reading this in a few months instead of now. Perhaps avoid things like this in our updates and word them more like e.g. VSCode updates: https://code.visualstudio.com/updates/v1_30

Reply:
A slightly new wording will appear in coming commit :)